bundles/action/org.openhab.action.ciscospark/OSGI-INF/action.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" configuration-policy="require" deactivate="deactivate" immediate="true" name="org.openhab.action.ciscospark.action">
```

Comment:
I do not know if specifying `configuration-policy` without also specifying `configuration-pid` works.  I think (but am not sure) that the absence of `configuration-pid` causes it to default to the value of the `name` attribute, which in this case is incorrect.  `configuration-pid` should be the same as the value of the `service-pid` property.  Together, `configuration-policy="require"` and `configuration-pid="org.openhab.ciscospark"` should mean that the component is only activated if there is a configuration under the persistence ID `org.openhab.ciscospark`.